kubejs/assets/kubejs/lang/pl_pl.json

```diff
@@ -0,0 +1,11 @@
+{
+  ///// FLUIDS NAME HERE /////
+  "fluid.kubejs.molten_calorite" : "Kaloryt",
+  "fluid.kubejs.molten_ostrum" : "Ostrum",
+  "fluid.kubejs.molten_desh" : "Desh",
```

Comment:
"Desz" would be a better name as this was the name used in Galacticraft Polish translation.

Reply:
I had no idea of the Galacticraft Polish translation as I have never touched that modpack in my life